### ISSUE TITLE
Execution environments no longer build

### ISSUE THREAD

**Issue by felixfontein** [ansible-community/images#98]
See https://github.com/ansible-community/images/actions/runs/11660326522/job/32462440567#step:5:65:
```
[1/4] STEP 13/14: RUN /output/scripts/pip_install $PYCMD
+ PYCMD=/usr/bin/python3
+ '[' -z /usr/bin/python3 ']'
+ '[' '!' -x /usr/bin/python3 ']'
/usr/bin/python3 is not an executable
+ echo '/usr/bin/python3 is not an executable'
+ exit 1
Error: error building at STEP "RUN /output/scripts/pip_install $PYCMD": error while running runtime: exit status 1
```
I guess it's now necessary to explicitly install Python 3 in the image.

**Comment by samccann** [comment]
This isn't related to https://github.com/ansible/ansible-documentation/issues/2093 is it?

**Comment by felixfontein** [comment]
Yep, that's exactly the same.

**Comment by Andersson007** [comment]
@anweshadas could you change the image in the builder's files to a working one?

**Comment by samccann** [comment]
Do we know the underlying problem here? (other than latest fedora doesn't work). What do we need to change so that  the latest fedora DOES work?

**Comment by felixfontein** [comment]
The underlying problem is likely that Python is no longer installed by default in the base image. Fixing that usually requires installing Python first. Adding something like `RUN $PKGMGR install -y python3` to `additional_build_steps.prepend_base` (https://ansible.readthedocs.io/projects/builder/en/stable/definition/#additional-build-steps) might help.

**Comment by Andersson007** [comment]
How about just fixing the version `fedora:latest` -> `fedora:39` for now? it doesn't really matter which one to use. It's imo better not to rely on latest anyway

**Comment by felixfontein** [comment]
There's a PR that fixes the situation with `fedora:latest`: #99.

**Comment by Andersson007** [comment]
> moving off latest means we have to come back in periodically and update to a maintained version of fedora here.

it's imo OK because we anyway need to fix stuff like this coming with `latest` periodically. Anyway, not advocating too much for bumping the version manually:)
